index.html

```diff
@@ -592,7 +592,7 @@ <h3>Use Cases and Requirements</h3>
 an essential part of processing a <a>verifiable credential</a>. At the time
 of publication, Working Group members had implemented
 <a>verifiable credentials</a> using at least three proof mechanisms:
-JSON Web Tokens [[RFC7519]], Linked Data Signatures [[?LD-SIGNATURES]], and
+JSON Web Signature [[RFC7515]], Linked Data Signatures [[?LD-SIGNATURES]], and
```

Comment:
Perhaps --
`JSON Web Tokens [[RFC7519]] with JSON Web Signatures [[RFC7515]]; Linked Data Signatures [[?LD-SIGNATURES]]; and`